ui-v2/app/styles/components/feedback-dialog/skin.scss

```diff
@@ -1,5 +0,0 @@
-%feedback-dialog-inline p,
-%feedback-dialog-inline p::after {
-  color: $ui-white;
-  background-color: $ui-gray-800;
-}
```

Comment:
I've kept this file here but empty for the moment:

1. For consistency
2. In case I need to add some sort of skin later

To delete or not to delete, that is the question..

Reply:
If it were me, I would delete it. It's a personal preference, but I don't like void things hanging around, whether its an empty function, a stale comment, or a file for the sake of consistency. It might be more confusing to leave it in there; someone may come along and think it has some purpose defined elsewhere.

Reply:
Cool, gone!